README.md

```diff
@@ -241,7 +241,39 @@ If multiple images are to be added, `modules` is an array, which holds the names
 to be added, in that particular order.
 
 optional_otions is just an optional parameter, in object form, which you might
-want to provide to the modules. 
+want to provide to the modules.
+
+A variety of syntaxes are supported by Image Sequencer to add multiple steps and configurations quickly for module chaining. This can be achieved by passing strings to `sequencer.addStep()`:
```

Comment:
We might say the two major ones are "string syntax" and JSON, where string syntax is designed to be as compact and URL-friendly as possible, and JSON is capable of more complex sequences.